nanover-rs/src/simulation.rs

```diff
@@ -496,6 +496,7 @@ impl OpenMMSimulation {
     unsafe fn add_imd_force(n_particles: i32) -> *mut OpenMM_CustomExternalForce {
         let energy_expression = CString::new("-fx * x - fy * y - fz * z").unwrap();
         let force = OpenMM_CustomExternalForce_create(energy_expression.into_raw() as *const i8);
+        OpenMM_Force_setForceGroup(force, 31);
```

Comment:
Not sure if `OpenMM_CustomExternalForce_setForceGroup` exists, or if somehow `force` can be cast to the correct struct type?